packages/ipfs-core/src/components/swarm/disconnect.js

```diff
@@ -16,7 +16,7 @@ module.exports = ({ network }) => {
    */
   async function disconnect (addr, options) {
     const { libp2p } = await network.use(options)
-    return libp2p.hangUp(addr, options)
+    return libp2p.hangUp(addr)
```

Comment:
Just to do the same thing you did with `dial`

```suggestion
    await libp2p.hangUp(addr)
```